logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc

```diff
@@ -0,0 +1,434 @@
+----
+
+.. Increase the max allocation retries:
++
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.allocation.max_retries":10}'
+----
+
+.. Delete all the scroll items:
++
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_search/scroll/_all -X DELETE
+----
+
+.. Increase the timeout:
++
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.unassigned.node_left.delayed_timeout":"10m"}'
+----
+
+. If the preceding steps do not clear the red indices, delete the indices individually.
+
+.. Identify the red index name:
++
+----
+oc exec -n openshift-logging -c <elasticsearch_pod_name> -- es_util --query=_cat/indices?v
+----
+
+.. Delete the red index:
++
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_red_index_name> -X DELETE
+----
+
+. If there are no red indices and the cluster status is red, check for a continuous heavy processing load on a data node.
+
+.. Check if the Elasticsearch JVM Heap usage is high:
++
+----
+oc exec -n openshift-logging -c <elasticsearch_pod_name> -- es_util --query=_nodes/stats?pretty
+----
++
+In the command output, review the `node_name.jvm.mem.heap_used_percent` field to determine the JVM Heap usage.
+
+.. Check for high CPU utilization.
+// TBD - how?
+
+[id="elasticsearch-cluster-health-is-yellow"]
+== Elasticsearch Cluster Health is Yellow
+
+Replica shards for at least one primary shard are not allocated to nodes.
+
+=== Troubleshooting
+
+. Check the disk space of the Elasticsearch node.
+
+. Increase the node count or the disk space of existing nodes by adjusting `nodeCount` or `storage.size` in the `ClusterLogging` CR.
+
+.Additional resources
+
+* See *Elasticsearch Disk Usage* in xref:../../../logging/cluster-logging-dashboards.adoc#cluster-logging-dashboards-logging_cluster-logging-dashboards[OpenShift Logging dashboards].
+// https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-dashboards.html#cluster-logging-dashboards-logging_cluster-logging-dashboards
+* xref:../../../logging/config/cluster-logging-configuring-cr.html[About the Cluster Logging custom resource]
+// https://docs.openshift.com/container-platform/4.7/logging/config/cluster-logging-configuring-cr.html
+* xref:../../../logging/config/cluster-logging-log-store.adoc#cluster-logging-elasticsearch-storage_cluster-logging-store[Configuring persistent storage for the log store]
+// https://docs.openshift.com/container-platform/4.7/logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-storage_cluster-logging-store
+
+[id="elasticsearch-write-requests-rejection-jumps"]
+== Elasticsearch Write Requests Rejection Jumps
+
+=== Troubleshooting
+TBD
```

Comment:
@sasagarw @ewolinetz  Please provide troubleshooting steps.

Reply:
@rolfedh `Elasticsearch Write Requests Rejection Jumps` is a `warning` alert and we haven't documented troubleshooting steps for warning alerts yet. I guess you can skip this for future releases.